test/unit/math/prim/arr/meta/is_constant_struct_test.cpp

```diff
@@ -5,7 +5,14 @@
 TEST(MetaTraits, isConstantStruct) {
   using stan::is_constant_struct;
   using std::vector;
-  EXPECT_TRUE(is_constant_struct<vector<double> >::value);
-  EXPECT_TRUE(is_constant_struct<vector<vector<double> > >::value);
-  EXPECT_TRUE(is_constant_struct<vector<vector<vector<double> > > >::value);
+  EXPECT_TRUE(is_constant_struct<vector<double>>::value);
```

Comment:
[optional]
I sometimes define helpers, like

```
<template typename... Ts>
void expect_is_const() {
  EXPECT_TRUE(is_constant_struct<Ts>::value);
}
```

and then the tests simplify to

```
expect_const<vector<double>>();
expect_const<vector<vector<double>>>();
...
```